src/map.h

```diff
@@ -609,7 +609,8 @@ class map
 // Flags: 3D
     std::string features( const tripoint &p ); // Words relevant to terrain (sharp, etc)
     bool has_flag( const std::string &flag, const tripoint &p ) const;  // checks terrain, furniture and vehicles
-    bool can_put_items( const tripoint &p ); // True if items can be placed in this tile
+    bool can_drop_items( const tripoint &p ) const; // True if items can be dropped in this tile
+    bool can_put_items( const tripoint &p ) const; // True if items can be placed in this tile
```

Comment:
> `can_put_ter_furn` suggests "can put on terrain or furniture but not on anything else"

Ahh, maybe `can_put_items_on_terrain_or_furniture` would be better? I'd at least have gleaned some of what you've explained from that, rather than `can_put_items_ter_furn`.